src/Form/EmbedButtonForm.php

```diff
@@ -168,6 +168,39 @@ protected function copyFormValuesToEntity(EntityInterface $entity, array $form,
   }
 
   /**
+   * Builds a list of entity type labels suitable for embed button options.
+   *
+   * Configuration entity types without a view builder are filtered out while
+   * all other entity types are kept.
+   *
+   * @return array
+   *   An array of entity type labels, keyed by entity type name.
+   */
+  protected function getFilteredEntityTypes() {
+    $options = array();
+    $definitions = $this->entityManager->getDefinitions();
+
+    foreach ($definitions as $entity_type_id => $definition) {
+      // Don't include configuration entities which do not have a view builder.
+      if ($definition->getGroup() != 'configuration' || $definition->hasViewBuilderClass()) {
```

Comment:
I thought that it might be confusing. The idea is following:
- Config entities with a view builder are allowed.
- All content entities, even without a view builder (like files).

So, based on that all those entities are allowed which are either "not config" or have a view builder. Hence, its an OR instead of AND.

Do you think it'll be more clear if I wrote `if ($definition->getGroup() == 'content' || $definition->hasViewBuilderClass()` instead?